common/endpoint-shared/src/main/java/org/kaaproject/kaa/common/avro/AvroJsonConverter.java

```diff
@@ -91,7 +91,8 @@ public T decodeJson(byte[] data) throws IOException {
      * @throws IOException Signals that an I/O exception has occurred.
      */
     public T decodeJson(String data, T reuse) throws IOException{
-        jsonDecoder = DecoderFactory.get().jsonDecoder(this.schema, data);
+        boolean base64 = true;
+        jsonDecoder = DecoderFactory.get().jsonDecoder(this.schema, data, base64);
```

Comment:
Don't declare base64 variable, just call jsonDecoder method with true as the last argument